bears/r/FormatRBear.py

```diff
@@ -1,22 +1,69 @@
 from coalib.bearlib.abstractions.Linter import linter
+from coalib.bearlib.spacing.SpacingHelper import SpacingHelper
 from coalib.parsing.StringProcessing import escape
 
 
+def _map_to_r_bool(py_bool):
+    return 'TRUE' if py_bool else 'FALSE'
+
+
 @linter(executable='Rscript',
         output_format='corrected',
         prerequisite_check_command=('Rscript', '-e', 'library(formatR)'),
         prerequisite_check_fail_message='Please install formatR for this bear '
                                         'to work.')
 class FormatRBear:
-
     """
     Check and correct formatting of R Code using known formatR utility.
     """
     LANGUAGES = "R"
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
+    def create_arguments(filename, file, config_file,
+                         r_keep_comments: bool=True,
+                         r_keep_blank_lines: bool=True,
+                         r_braces_on_next_line: bool=False,
+                         r_use_arrows: bool=False,
+                         tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH,
+                         r_max_expression_length: int=20):
```

Comment:
is there an option to disable this feature? I would not expect to get something like (1 + 1 + 1) and so on wrapped if it's not because of the line length^^